mapswipe_workers/sample_data/change_detection_tutorial_bing.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Change Detection Tutorial",
+  "projectType": 3,
+  "projectId": "change_detection_tutorial",
+  "zoomLevel": 16,
+  "lookFor": "CHANGES IN BUILDINGS",
+  "status": "change_detection_tutorial",
+  "projectDetails": "This is a tutorial",
+  "progress": 0,
+  "contributorCount": 0,
+  "tileServerA" : {
+    "apiKey" : "AopsdXjtTu-IwNoCTiZBtgRJ1g7yPkzAi65nXplc-eLJwZHYlAIf2yuSY_Kjg3Wn",
+    "captions" : "",
+    "date" : "",
+    "name" : "bing",
+    "url" : "https://ecn.t0.tiles.virtualearth.net/tiles/a{quad_key}.jpeg?g=7505&mkt=en-US&token={key}",
+    "credits": "© 2019 Microsoft Corporation, Earthstar Geographics SIO"
+  },
+  "tileServerB" : {
+    "apiKey" : "91e57457-aa2d-41ad-a42b-3b63a123f54a",
+    "captions" : "",
+    "date" : "",
+    "name" : "maxar_premium",
+    "url" : "https://services.digitalglobe.com/earthservice/tmsaccess/tms/1.0.0/DigitalGlobe%3AImageryTileService@EPSG%3A3857@jpg/{z}/{x}/{y}.jpg?connectId={key}",
+    "credits": "© 2019 Maxar"
+  },
+  "examplesFile": "sample_data/change_detection_tutorial_tasks_bing.geojson",
+  "categories": {
+    "changes_1": {
+      "pre": "For this project we are looking for new settlements. Compare the upper (before) and lower (after) image and *SWIPE RIGHT* (not tap) to indicate that new settlements have been built.",
+      "post_correct": "Yes, that looks like a new settlement being developed. Plenty of new houses and huts have been built in this area. Swipe left to continue the tutorial.",
+      "post_wrong": "Oh no. Make sure to spot all the new settlements and swipe right. New roads can also function as an indicator sometimes."
+    },
+    "changes_2": {
+      "pre": "And again: If new settlements have been built, *SWIPE* to the right side.",
+      "post_correct": "Correct. We can spot new settlements in the upper right part of the satellite image. Swipe left to go on with some more examples.",
+      "post_wrong": "Take a closer look. There are new settlements here in the upper right part of the satellite image."
+    },
+    "no_changes_1": {
+      "pre": "In this example, there are many buildings but *NO NEW SETTLEMENTS* emerged. *SWIPE* left to indicate no changes.",
```

Comment:
I think it's worth adding a step where buildings have disappeared, to say they do count as changes (unless this is a per project decision?). It was one of the first questions I got during user testing yesterday